### ISSUE TITLE
Search Tabs button doesn't work with Touch UI Layout enabled

### ISSUE THREAD

**Issue by archerallstars** [brave/brave-browser#33071]
## Description 

Search Tabs button (Ctrl+Shift+A) doesn't work with Touch UI Layout enabled. Clicking on it has no effect at all. I have attached the screen recording to this report.

## Steps to Reproduce

   1. Go to `brave://flags`.
   2. Enable the Touch UI Layout.
   3. Reluanch.
   4. Click on the Search Tabs button (Ctrl+Shift+A).
   5. The search tabs menu won't show up.

## Actual result:

https://github.com/brave/brave-browser/assets/1403194/84a9f21f-d16e-4d4d-a390-34235d304ceb

## Expected result:
When clicking on the Search Tabs button(Ctrl+Shift+A), the search tabs menu should appear.

## Reproduces how often: 

Always.

## Brave version (brave://version info)

```
Brave | 1.58.124 Chromium: 117.0.5938.62 (Official Build) (64-bit)
-- | --
Revision | 5c0e98018fb16948401122cb9af1a5a0a6b40865
OS | Linux
```

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes.
- Can you reproduce this issue with the beta channel? Yes, as of Brave 1.59.90 Chromium: 117.0.5938.88 (Official Build) beta (64-bit), Revision 5272112c86588e44ec7c0b766c318cddccbced85.
- Can you reproduce this issue with the nightly channel? Yes, as of Brave 1.60.30 Chromium: 117.0.5938.88 (Official Build) nightly (64-bit), Revision 51564671a5c657bcf2633e5d26878d3633c6b338.

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No.
- Does the issue resolve itself when disabling Brave Rewards? No.
- Is the issue reproducible on the latest version of Chrome? No.

## Miscellaneous Information:

I am using the platform Ozone Wayland option on GNOME 44.4 with fractional scaling at 125%.

![Screenshot from 2023-09-17 01-23-58](https://github.com/brave/brave-browser/assets/1403194/725d3c6c-b5d4-4c17-917c-3d6bce4a32bc)

**Comment by kreaninw** [comment]
This issue is fixed in the nighty channel (Version 1.61.32 Chromium: 118.0.5993.88), as shown in the screenshot below:

![Screenshot from 2023-10-16 06-51-54](https://github.com/brave/brave-browser/assets/1403194/aca47b29-bd71-4c2a-bc5a-f9f47f626b98)

 The search tabs button is replaced with a magnifier icon, and the click area is now fixed. This issue will be closed when this fix lands on the stable channel.

**Comment by kreaninw** [comment]
This issue is fixed in the nighty channel (Version 1.61.32 Chromium: 118.0.5993.88), as shown in the screenshot below:

![Screenshot from 2023-10-19 09-45-28](https://github.com/brave/brave-browser/assets/1403194/74ee3693-f76a-4799-b6d4-dfec47793c5a)

 The search tabs button is replaced with a manifier ico

**Comment by archerallstars** [comment]
It seems, the fix in the nightly release isn't back ported to the current stable release: version 1.62.153 Chromium: 121.0.6167.85 (Official Build) (64-bit).

Is there any progress regarding this issue?

**Comment by archerallstars** [comment]
The issue is fixed, as of Brave 1.62.156, as shown in the screenshot below:

![Screenshot](https://github.com/brave/brave-browser/assets/1403194/d7d3c6f3-9e15-4fc8-81e2-961879f00da0)
